### DISCUSSION TITLE
preciso de ajuda kk

### DISCUSSION THREAD

**Discussion by Ingrid-end** [frontendbr/forum#2103] [Perguntas e Respostas]
Estou com um probleminha, por alguma razão tem uma faixa branca na minha pagina, e não sei daonde ela veio, mas não quer ir embora .. 

https://cdn1.gnarususercontent.com.br/1/749309/2a30c190-c326-4abf-ae54-c40f2e627b44.png
 
Esse em vermelho é o footer, o azul é o body normal, ai tem esse espaço em branco

**Comment by Kaedh** [comment]
No caso esse body deveria ocupar todo o espaço disponível até chegar no footer?
tenta dá uma olhada em como você definiu os height deles, se possível posta o código aqui pra gente :)

**Reply by Ingrid-end** [reply]
Oii, então, o codigo é bem grande e o css tbm, mas eu percebi uma coisa, essa faixa so aparece quando tem tabela, o restante das paginas estão normais.
 Sobre o body: 
html{
/*   background: rgb(255,255,255);
background: linear-gradient(#ffffff,#c5d9d9); */
}
body{
  background-attachment:fixed;
  background-origin:inital;
  background-repeat:no repeat;
  background: rgb(255,255,255);
  background: linear-gradient(#ffffff,#c5d9d9);}
 
 ( Não tem tamanho ... Isso pode prejudicar?!)

Footer:
footer {
   font-size: 80%;
   opacity: 50%;
   text-align: right;
   font-family: 'rubik';
   bottom:0;
   position: static;
   right: 0;
   background-color: red;
   }
 ( o footer é um include, não sei se isso tbm muda alguma coisa)

**Reply by Kaedh** [reply]
Eu tentaria colocar a altura do tag html e da tag body como 100%  (ou 100vh oq é até melhor), assim o body vai tentar preencher 100% da altura disponivel.

**Reply by Ingrid-end** [reply]
Blz, vo testar aqui, obrigada, se tu quiser ainda o codigo, é que ele tem 488 linhas kkk
 :)

**Reply by Kaedh** [reply]
Vish kkk, testa ai e vê se funciona!

**Reply by Ingrid-end** [reply]
Acheii kkk, vlw pela ajuda, era o tamanho de uma div grid da s tabelas ...
